controllers/productController.js

```diff
@@ -0,0 +1,12 @@
+const { json } = require("express");
+const { catchAsync } = require("../");
+const { productService } = require("../services/productService");
+
+const getProductsById = async (req, res) => {
+  const product = await productService(req.params.productId);
```

Comment:
- product의 id 값을 할당하는 부분이기 때문에 변수명도 이에 맞게 수정해주세요!
- service 단에서 함수명이 변경된 것 같은데 확인하여 수정해주시면 될 것 같습니다!

```suggestion
  const productId = await getProductById(req.params.productId);
```